low-level/framework/rabbitmq/rabbitmq_egress_accumulated_msgs_processor.py

```diff
@@ -109,33 +93,38 @@ def run(self):
             # Check for shut down message from sspl_ll_d and set a flag to shutdown
             #  once our message queue is empty
             self._jsonMsg, _ = self._read_my_msgQ()
-            if self._jsonMsg.get("message").get("actuator_response_type") is not None and \
-                self._jsonMsg.get("message").get("actuator_response_type").get("thread_controller") is not None and \
-                self._jsonMsg.get("message").get("actuator_response_type").get("thread_controller").get("thread_response") == \
+            if self._jsonMsg.get("message").get(
+                    "actuator_response_type") is not None and \
+                    self._jsonMsg.get("message").get(
+                        "actuator_response_type").get(
+                        "thread_controller") is not None and \
+                    self._jsonMsg.get("message").get(
+                        "actuator_response_type").get("thread_controller").get(
+                        "thread_response") == \
                     "SSPL-LL is shutting down":
-                    logger.info("RabbitMQEgressAccumulatedMsgsProcessor, run, received" \
-                                    "global shutdown message from sspl_ll_d")
-                    self.shutdown()
+                logger.info(
+                    "RabbitMQEgressAccumulatedMsgsProcessor, run, received"
+                    "global shutdown message from sspl_ll_d")
+                self.shutdown()
         try:
+            # TODO : Fix accumulated message processor when message bus changes are available to
```

Comment:
You may add foundation team tkt number also in TODo

Reply:
Done